main.py

```
@@ -270,6 +334,60 @@ def main():
     # Add event listener for page load
     window.events.loaded += on_window_loaded
     
+    # Create a local server to serve static files like version.txt
+    import http.server
+    import socketserver
+    import threading
+    import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)